shared-bindings/displayio/Bitmap.c

```diff
@@ -172,6 +172,115 @@ STATIC mp_obj_t bitmap_subscr(mp_obj_t self_in, mp_obj_t index_obj, mp_obj_t val
     return mp_const_none;
 }
 
+//|     def blit(self, x: int, y: int, source_bitmap: bitmap, x1: int, y1: int, x2: int, y2: int, skip_index: int) -> Any:
```

Comment:
```suggestion
//|     def blit(self, x: int, y: int, source_bitmap: bitmap, *, x1: int, y1: int, x2: int, y2: int, skip_index: int) -> Any:
```

The * indicates everything after it is KW_ONLY (which it is and should be.)

Reply:
Updated per requested change.